service/src/main/java/org/cbioportal/service/impl/GenericAssayServiceImpl.java

```diff
@@ -80,15 +80,15 @@
                 }
             } else {
                 List<GenericAssayAdditionalProperty> additionalProperties = genericAssayRepository.getGenericAssayAdditionalproperties(distinctStableIds);
+                Map<String, List<GenericAssayAdditionalProperty>> additionalPropertiesGroupedByStableId =
```

Comment:
Line 77: `if (projection == "ID") {` - that can't be right, can it?

Reply:
This is not correct... updated